grml-debootstrap

```diff
@@ -951,26 +1025,29 @@ fi
 }
 # }}}
 
-# format efi partition {{{
-format_efi_partition() {
-  if [ -z "$EFI" ] ; then
-    return 0
-  fi
-
-  if ! [ -b "$EFI" ] ; then
-    eerror "Specified efi argument [$EFI] not a valid block device."
-    bailout 1
-  fi
-
-  if fsck.vfat -bn "$EFI" >/dev/null; then
-    einfo "EFI partition $EFI seems to have a FAT filesystem, not modifying."
-  else
-    einfo "EFI partition $EFI doesn't seem to be formatted, creating filesystem."
-    if ! mkfs.fat -F32 -n "EFI" "$EFI" ; then
-      eerror "Error while creating filesystem on ${EFI}."
+# format boot info partitions (ESP, Raspberry Pi firmware) {{{
+format_bootinfo_partitions() {
```

Comment:
other tools tend to call them "firmware boot partitions"

Reply:
Renamed.